### DISCUSSION TITLE
Move requests between collections

### DISCUSSION THREAD

**Discussion by guydav1** [thunderclient/thunder-client-support#1658] [Ideas]
Need some interface to move a selected request from collection A to B.
currently if I need the same request in both collections, I will duplicate it in A, and slowly drag it across all other requests / collections to insert in to B.
some "Move request" button which opens a select menu with all the available collection would be helpful

**Comment by rangav** [comment]
Hi @guydav1 can you please confirm - are you using the free or paid version?

**Reply by guydav1** [reply]
paid business version

**Reply by rangav** [reply]
workaround:
If you are using the database format v4, then you can move the request files easily and restart vs code
https://docs.thunderclient.com/team#database-format-v4
